features/bevragen/persoon/verblijfplaats/onbekend/protocollering-antwoord.feature

```diff
@@ -0,0 +1,34 @@
+#language: nl
+
+Functionaliteit: protocollering van het antwoord voor verblijfplaats onbekend
+
+  Rule: In het antwoord geleverde velden worden geprotocolleerd als de elementnummers volgens Logisch ontwerp BRP
+    Dit is een 6-cijferige code, met zo nodig voorloopnul voor categorieën.
+    Alleen velden die echt geleverd zijn, dus een waarde hebben in de response, worden opgenomen als geleverde rubriek
+    Ook alle velden die worden meegeleverd voor het bepalen van het verblijfplaats type worden opgenomen.
+    
+    Abstract Scenario: Protocolleren van geleverde velden voor <fields veld> voor een persoon met onbekende verblijfplaats
```

Comment:
Volgens mij maakt het voor de Dan stap niet uit of in onderstaand scenario een onbekend verblijfplaats op basis van land='0000' of op basis van naamOpenbareRuimte='.' wordt vastgesteld. Volgens mij worden dan dezelfde rubrieknummers vastgelegd, toch?

Reply:
we hebben afgesproken dat type onbekend alleen wordt bepaald op land=0000. Type wordt niet bepaald op naamOpenbareRuimte, maar op straat. Heeft die een standaardwaarde ".", dan is het type toch Adres.

Reply:
Mag je dan stellen dat land=0000 betekent dat de verblijfplaats over het algemeen onbekend is of alleen dat de verblijfplaats in het buitenland onbekend is?
In ons overleg van 7 december is nl. dat laatste gesteld.